src/Microsoft.Health.ControlPlane.Core.UnitTests/Microsoft.Health.ControlPlane.Core.UnitTests.csproj

```diff
@@ -1,21 +1,10 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
-  <PropertyGroup>
-    <TargetFramework>netcoreapp2.2</TargetFramework>
-    <IsPackable>false</IsPackable>
```

Comment:
Do we want to keep this?

Reply:
<IsPackable>false</IsPackable> is set in Directory.Build.props because the filename contains "Test"